tests/performance/test_data_analysis/conftest.py

```diff
@@ -0,0 +1,132 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+"""
+Pytest configuration for Statistical Data Analysis tests
+
+This module contains fixtures that allow to obtain the test input data and to manage the final report.
+
+Functions:
+    - pytest_addoption: defines the input parameters when running the test.
+    - get_data: fixture that gets the CSV files and the confidence level passed as parameters.
+    - config: fixture that gets the YAML file with the configuration of the test.
+    - pytest_runtest_makereport: generates the necessary information in the final test report.
+"""
+
+import os
+import pytest
+import pytest_html
+
+from typing import Generator, Tuple
+from statistical_data_analyzer import DataLoader
```

Comment:
```suggestion
import os
from typing import Generator, Tuple

import pytest
import pytest_html

from statistical_data_analyzer import DataLoader
```

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5622/commits/a8be0593badbbd04bb3211468f64ad2689d17bf3